src/Entity/Adherent.php

```diff
@@ -1051,6 +1051,11 @@ public function isAdherent(): bool
         return $this->adherent;
     }
 
+    public function isUser(): bool
+    {
+        return !$this->adherent;
```

Comment:
It would be better to call the `isAdherent` method not to break encapsulation of this behavior.

Reply:
I think it's more readable to have an explicit `isUser()` method to check if the person is an user.
Because everywhere you call the method `!$object->isAdherent()`, you will have to explain why it is a user.. it's not really logic.

Reply:
> I think it's more readable to have an explicit isUser() method to check if the person is an user.
I agree

My point was to keep the method but use the method `isAdherent()` instead of the property directly.

Reply:
Ah ok, nice idea :+1: